SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -886,11 +899,6 @@ void USpatialSender::SendCreateEntityRequest(USpatialActorChannel* Channel)
 	Receiver->AddPendingActorRequest(RequestId, Channel);
 }
 
-void USpatialSender::SendDeleteEntityRequest(Worker_EntityId EntityId)
```

Comment:
This method has been renamed RequestEntityDeletion to indicate via the API that we are no longer guaranteed a deletion will take place.  The new method, RequestEntityDeletion, will switch internally based on whether or not the deletion is for a startup actor.  In the case it is, it will flag the tombstone component.  Otherwise it will proceed as normal with a SendDeleteEntityRequest.

RequestEntityDeletion additionally has a boolean flag allowing us to force entity deletion.  This can be used specifically for cases where the NetDriver (for example) is being torn down and we want to force deletion of a specific entity.

We may want to tackle larger API changes at some point to solve a variety of other unrelated issues:
 - Some callers use SpatialSender::SendDeleteEntityRequest(), while others use the method of the same name on the Connection directly.
 - Sender and Receiver call back and forth into one another.  In at least some of these cases the functionality is not really specific to the sender or receiver, and could be moved to a separate shared location (ex. SpatialReceiver::RemoveActor())